test/appium/views/chat_view.py

```diff
@@ -186,6 +187,22 @@ def wait_for_messages_by_user(self, username: str, expected_messages: list, erro
             errors.append('Not received messages from user %s: "%s"' % (username, ', '.join(
                 [i for i in list(set(expected_messages) - set(received_messages))])))
 
+    def wait_for_messages(self, username: str, expected_messages: list, errors: list, wait_time: int = 30):
+        expected_messages = expected_messages if type(expected_messages) == list else [expected_messages]
+        repeat = 0
+        received_messages = list()
+        while repeat <= wait_time:
```

Comment:
How about implementing a `wait_until_present(element, wait_time)` method that could be reused? It could be based on `selenium.webdriver.support.ui.WebDriverWait` and would throw `TimeoutException`.

Reply:
Or maybe just use `wait_for_element`?

Reply:
We don't need to raise exception right here, we do need to check presence/absence of all sent messages